activetable/static/js/src/activetable.js

```diff
@@ -0,0 +1,96 @@
+/* Javascript for ActiveTableXBlock. */
+function ActiveTableXBlock(runtime, element, init_args) {
+
+    var checkHandlerUrl = runtime.handlerUrl(element, 'check_answers');
+    var saveHandlerUrl = runtime.handlerUrl(element, 'save_answers');
+
+    function markResponseCells(data) {
+        if (data.answers_correct) {
+            $.each(data.answers_correct, function(cell_id, correct) {
+                var $cell = $('#' + cell_id, element);
+                $cell.removeClass('right-answer wrong-answer unchecked');
+                if (correct) {
+                    $cell.addClass('right-answer');
+                    $cell.prop('title', 'correct');
+                } else {
+                    $cell.addClass('wrong-answer');
+                    $cell.prop('title', 'incorrect');
+                }
+            });
+        } else {
+            $('td.active', element).removeClass('right-answer wrong-answer').addClass('unchecked');
+        }
+    }
+
+    function updateStatusMessage(data) {
+        var $status = $('.status', element);
+        var $status_message = $('.status-message', element);
+        if (!data.answers_correct) {
+            $status.removeClass('incorrect correct');
+            $status.text('unanswered');
+            $status_message.text('');
+        }
+        else if (data.num_total_answers == data.num_correct_answers) {
+            $status.removeClass('incorrect').addClass('correct');
+            $status.text('correct');
+            $status_message.text('Great job!');
+        } else {
+            $status.removeClass('correct').addClass('incorrect');
+            $status.text('incorrect');
+            $status_message.text(
+                'You have ' + data.num_correct_answers + ' out of ' + data.num_total_answers +
+                ' cells correct.'
```

Comment:
The current recommendation seems to be to mark string in Python files, but not anywhere else.